lua/nvim-lsp-installer/servers/rescriptls/init.lua

```diff
@@ -0,0 +1,23 @@
+local server = require("nvim-lsp-installer.server")
+local path = require("nvim-lsp-installer.path")
+local shell = require("nvim-lsp-installer.installers.shell")
+
+local root_dir = server.get_server_root_path('rescriptls')
+
+return server.Server:new {
+    name = "rescriptls",
+    root_dir = root_dir,
+    installer = installers.when {
+        unix = shell.bash [[
+           curl -s https://api.github.com/repos/rescript-lang/rescript-vscode/releases/latest \
```

Comment:
Let's add `-f` so we fail early and don't pipe potential error responses

```suggestion
           curl -fs https://api.github.com/repos/rescript-lang/rescript-vscode/releases/latest \
```